lib/console.js

```diff
@@ -249,6 +249,75 @@ Console.prototype.groupEnd = function groupEnd() {
     this[kGroupIndent].slice(0, this[kGroupIndent].length - 2);
 };
 
+const untablableTypes = [
+  'symbol', 'undefined', 'boolean', 'number', 'string', 'function',
+];
+
+const untablableValues = [
+  undefined, null, NaN, true, false, global, process,
```

Comment:
Why are `global` and `process` singled out?

Reply:
they are objectively easier to read when not run through the table method, and the spec says `Fall back to just logging the argument if it can’t be parsed as tabular.` which i think is fairly true for both of those.

Reply:
I agree in this case that it is going to be easier to read. The question is if we want to have a special handling here. And there could theoretically be other things that are difficult to read as well because they have lots of properties.

In chrome `window` works as tableable and I think that is comparable. I guess most people will not add `process` or `global` so I would say either we should not special handle them at all or we add a limit that by checking the property number.